README.md

```diff
@@ -30,6 +30,8 @@ Developing the UI has the same prequisites as trying it out. Once you've created
 pnpm start
 ```
 
+and open [`localhost:3000`](http://localhost:3000).
```

Comment:
Added because the 127.0.0.1 address displayed in the terminal results in CORS errors:

<img width="782" alt="image" src="https://user-images.githubusercontent.com/251288/192443818-4b2f4099-e49c-4a33-b4ac-59f91761b41c.png">

And I wasn't able to get vite to change it. I tried `--host localhost` and this:

![image](https://user-images.githubusercontent.com/251288/192443923-20b51b87-3ddb-4b74-b15b-aaabca56da14.png)
![image](https://user-images.githubusercontent.com/251288/192443943-34daaf0b-087e-48fd-b28e-65a444017858.png)